openedx/stanford/common/djangoapps/enrollment/views.py

```diff
@@ -160,7 +160,7 @@ def delete(self, request, course_id):
                 },
             )
         email_students = request.data.get('email_students', False) in ['true', 'True', True]
-        auto_enroll = request.data.get('auto_enroll', False) in ['true', 'True', True]
+        auto_enroll = request.data.get('auto_enroll', True) in ['true', 'True', True]
         email_params = {}
         language = None
         if email_students:
```

Comment:
Looking more widely at this code, it's a bit inefficient to query the User table twice just to get a single user. You can do it with a single db hit like so:

try:
    user = User.objects.get(email=email)
except User.DoesNotExist:
    pass
else:
    language = get_user_email_language(user)

Reply:
@stvstnfrd You can close this PR in favour of my one.